parse.go

```diff
@@ -750,10 +763,16 @@ func (p *Parser) process(args []string) error {
 		}
 
 		if spec.required {
+			if spec.envOnly {
```

Comment:
I think it's better to leave this as in @padilo's original PR, where this line was:

    if spec.short == "" && spec.long == "" {

Reply:
:heavy_check_mark: